runner/run_query.py

```diff
@@ -320,9 +322,10 @@ def ssh_spark(command):
   # Two modes here: Spark SQL Mem and Spark SQL Disk. If using Spark SQL disk use
   # uncached tables. If using Spark SQL Mem, used cached tables.
 
-  query_list = "set spark.sql.codegen=true; set spark.sql.shuffle.partitions = %s;" % opts.reduce_tasks
-  # Comment out for uncompressed output.
-  query_list += "SET hive.exec.compress.output=True;SET io.seqfile.compression.type=BLOCK;"
+  query_list = "set spark.sql.codegen=true;set spark.sql.shuffle.partitions=%s;" % opts.reduce_tasks
+  if (opts.compress):
```

Comment:
We could do that, except currently run_query.py does not know the input format. We'd either need to add a new flag "--file-format" like in prepare_benchmark.py, or add logic to determine the input format.